options/locale/locale_en-US.ini

```diff
@@ -1888,7 +1888,11 @@ settings.hooks = Webhooks
 settings.githooks = Git Hooks
 settings.basic_settings = Basic Settings
 settings.mirror_settings = Mirror Settings
-settings.mirror_settings.docs = Set up your project to automatically push and/or pull changes to/from another repository. Branches, tags, and commits will be synced automatically. <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/repo-mirror/">How do I mirror repositories?</a>
+settings.mirror_settings.docs = Set up your project to automatically push and/or pull changes to/from another repository. Branches, tags, and commits will be synced automatically. <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/usage/repo-mirror/">How do I mirror repositories?</a>
+settings.mirror_settings.docs.no_new_pull_mirrors = Set up your project to automatically push changes to another repository. Branches, tags, and commits will be synced automatically. <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/usage/repo-mirror/#pushing-to-a-remote-repository">How do I mirror repositories?</a>
+settings.mirror_settings.docs.no_new_push_mirrors = Push mirrors are currently disabled on your instance.
+settings.mirror_settings.docs.no_new_mirrors = Your repository is mirroring changes to/from another repository. Please keep in mind that you can't create any new mirrors at this time, however, you may still modify your settings here. <a target="_blank" rel="noopener noreferrer"https://docs.gitea.io/en-us/repo-mirror/">How do I mirror repositories?</a>
+settings.mirror_settings.docs.pull_mirror_instructions = To set up a pull mirror, please consult <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/usage/repo-mirror/#pulling-from-a-remote-repository">"Pulling from a remote repository" section</a>.
```

Comment:
These will invalidate all current translations. I think we better move at minimum the URL out of the translations, ideally the whole `<a>` tag.

Reply:
Some ideas for this in https://github.com/go-gitea/gitea/pull/24426#discussion_r1181111566, but nothing concrete yet on how we should do this.

Reply:
> These will invalidate all current translations.

I get that it's effort and that extra effort from the translators would be required, but intentional. I'm assuming that the TM would help here.

I am not sure if you are saying that "invalidating all current translations is unacceptable" or that we should should take advantage of the opportunity to remove the `<a>` tag from the translations.

> I think we better move at minimum the URL out of the translations, ideally the whole <a> tag.

My change proposes that we redirect the user directly to the right section. `en-us` is another problem, which is why I didn't do that in the first place... Are there other examples of that happening?

Reply:
Thanks, I see you moved out the URLs, which is great.

My point is, take a look at percentages on https://crowdin.com/project/gitea. Some of these languages will probably take a very long time to receive an update, if at all. So if they had translated the old links in the past, they could stay on them for years to come in theory.

Reply:
Got it. Are you sure that it's going to be i18n-able?